### ISSUE TITLE
Map objects' "Support" field

### ISSUE THREAD

**Issue by cbutcosk** [american-art/ccma#27]
Objects' "Support" field should be mapped, with the same outbound `crm:P45_consists_of` to a thesaurus term that the "Medium" field also uses currently.

**Comment by workergnome** [comment]
@cbutcosk:  Do you think of the support as a Material or as a Part?

Our assumption is that it's a part, because it can have dimensions and such—of course, that Part can have a material that it's made of.    

Otherwise, it's just an abstract type, and we can't talk any more about the support.

**Comment by cbutcosk** [comment]
Ah! That is much better actually, yeah. It definitely works for most (all?) of Colby's objects and would be clearer about our collages' material descriptions. Eg, the catalogued data for http://data.americanartcollaborative.org/page/ccma/object/4633 and http://data.americanartcollaborative.org/page/ccma/object/4634 read "AAA paper and BBB paper on paper," which otherwise would just result in `<http://..../4633> crm:P45_consists_of <http://.../thesaurus/paper>`.

How would you express the "on-ness" of a support? Is the fact that the watercolored paper is pasted to a sheet expressed by predicates between the parts?

edit: typo

**Comment by workergnome** [comment]
Here's @azaroth42's answer to that—he and I have been working on writing these patterns down (mostly him.  I'm just a sounding board.)

http://linked.art/model/object/physical/#parts

**Comment by azaroth42** [comment]
A crucially important sounding board for my mediocre writing! I'm just a scribe.

**Comment by cbutcosk** [comment]
Woah, that is very interesting you guys. It seems like Parts may be a good expression of plate v. sheet for prints?

I imagine a lot of museums may already record those dimensions-- http://data.americanartcollaborative.org/page/ccma/object/11 or http://data.americanartcollaborative.org/page/ccma/object/12 are examples in our data, but print cataloguers are pretty hardcore everywhere. It'd be interesting to use that for finding the plates used to strike a print, or states of one plate across collections (+1 for the "print scholars need LOD" thesis).

**Comment by workergnome** [comment]
Yeah—there's a lot of use in that level of detail. That's a lot of what we're striving for here—a pattern that works for simple cases, but that can go deeper as people's needs (and data)  become more complex.

**Comment by cbutcosk** [comment]
Definitely!

But is this a change we'll see in this phase of the AAC? If not, the CCMA's mapping needs amended to the `Disp_Material` field, instead of the `Material` field.

edit: nvm, it's already mapped that way. too little pre-github coffee.

**Comment by workergnome** [comment]
No worries!

**Comment by bsnikhila** [comment]
Just to be clear, I don't have to change anything in the mapping for 'Support', do I?

**Comment by cbutcosk** [comment]
No, you're good--pretty sure this would be a next-version thing.

**Comment by bsnikhila** [comment]
Great, thanks!
